doc/dimensionalityreduction/readme.md

```diff
@@ -0,0 +1,45 @@
+## Dimensionality reduction
+
+For getting more insights into your data, you can reduce the dimensionality of the measurements, using this algorithm:
+
+* [Uniform Manifold Approximation Projection (UMAP)](https://umap-learn.readthedocs.io/en/latest/)
+
+To apply it to your data, use the menu `Plugins > Compute Feature > Dimensionality reduction > UMAP`.
+Select the table whose features should be dimensionality reduced. Currently, the Spot and the BranchSpot table are
+supported.
+Next, select the feature + feature projections that should be dimensionality reduced.
```

Comment:
```suggestion
Next, select the feature + feature projections that should be dimensionality reduced. Select only features which describe the phenotype (spot size, shape, speed, number of neighbors, etc.) and avoid selecting technical features (e.g. Spot ID). Only select positional features (centroid) if the position of cells within the image are descriptive for the phenotype.
```

Just some ideas how to guide users. If they cannot select Spot ID anyway, please remove it from the suggestion.